test/unittests/state_bloom_test.cpp

```diff
@@ -0,0 +1,64 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "../utils/utils.hpp"
+#include <gtest/gtest.h>
+#include <test/state/bloom.hpp>
+#include <test/statetest/statetest.hpp>
```

Comment:
This include looks suspicious.

Reply:
Better now?

Reply:
No. Why do you need to include something from `test/statetest` when the tests are for `test/state`.

Reply:
done